backend/service/chaos/experimentation/experimentstore/experiment_store_test.go

```diff
@@ -169,74 +169,84 @@ func TestStopExperiments(t *testing.T) {
 	}
 }
 
-var getExperimentsTests = []struct {
```

Comment:
make this test less generic - current generic implementation adds a lot of code that's never evaluated since  the list of experiment tests has only 1 element. We can make it more complex later on if we decided that we need more complexity here.